### PR TITLE
Set default 'Platform' to AnyCPU for csccore and vbccore.

### PR DESCRIPTION
**Pull request by Sridhar-MS**
Fixes - #8951 

I am not sure if master branch is the right place to do this change. Please let me know if this PR needs to be done against another branch.

cc: @piotrpMSFT @jaredpar @agocke

### CODE REVIEW

File: src/Compilers/CSharp/CscCore/CscCore.csproj [hunk 1]
```diff
@@ -52,6 +52,26 @@
     <PlatformTarget>x64</PlatformTarget>
     <ErrorReport>prompt</ErrorReport>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'">
+    <DebugSymbols>true</DebugSymbols>
```
Comment (agocke) [inline src/Compilers/CSharp/CscCore/CscCore.csproj]: Many of these properties are incorrect. Also, I don't think we need to add a configuration, but change the existing configuration to be AnyCPU.

**Comment by jaredpar** [comment]
The title of this change is adding AnyCPU but it actually adds AnyCpu and a lot of unrelated settings.  Please limit this change to the scope of the title: changing AnyCPU.

**Comment by Sridhar-MS** [comment]
@jaredpar @agocke I just was trying to copy the existing property values for `x64` into `AnyCPU`. If you think that is not the right thing to do here I can remove the `x64` propertygroup and add `AnyCPU` propertygroup (making it same as [desktop csc project](https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/csc/csc.csproj#L31)) without any property values.

**Comment by agocke** [comment]
@Sridhar-MS Yes, I think mirroring the desktop csc is the right approach here.

**Comment by Sridhar-MS** [comment]
Fixed now and also squashed the commits.

**Comment by jaredpar** [comment]
:+1:

**Comment by agocke** [comment]
:+1:

**Comment by Sridhar-MS** [comment]
@dotnet-bot test this please
Requeuing tests since Jenkins was down.

**Comment by jaredpar** [comment]
Close / Open to see if Jenkins will wake up and see this.

**Comment by Sridhar-MS** [comment]
Can this be merged now?

**Pull request merged**